app/services/BulletPointBuilder.scala

```diff
@@ -330,11 +332,52 @@ object BulletPointBuilder {
   }
 
   def matchPhrases(p1: Phrase, p2: Phrase): Boolean = {
```

Comment:
uneeded braces

Reply:
I've put the methods explicitMatch and implicitMatch inside matchPhrases so the brackets are now necessary.